frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/details-card/card.tsx

```diff
@@ -30,57 +37,68 @@ const k8sResources: FirehoseResource[] = [
   },
 ];
 
-const DetailsCard = React.memo(
-  withDashboardResources((props) => {
-    const { watchK8sResource, stopWatchK8sResource, resources } = props;
+export const DetailsCard: React.FC<DashboardItemProps> = ({
```

Comment:
Why this change? This does not look like a refactor. More like changes in aesthetics.